data/themes/darktable.css

```diff
@@ -1537,10 +1537,11 @@ margin: 0 2px;
 #basics-box-labels #module-enable-button,
 #basics-link
 {
-  min-height: 0.95em;
-  min-width: 0.95em;
-  margin: 0 0.75em;
+  min-height: 1.15em;
+  min-width: 1.15em;
```

Comment:
To revert back.

Reply:
Just do a comparison with master. The sizes of the actual icons are identical between my PR (latest commit) and master. I'm telling you that the padding added by this change reduces the size of the icons. All I'm doing is compensating for that reduction.

Reply:
The only think I can conclude here is that there are some boundaries in em where things like icons change by whole pixel values. With my change, at font sizes 11 and 12 I agree the icons look a little bigger. With smaller font sizes (10 or below), the icons look the same to me. I don't know how to come up with a good compromise here.

Reply:
That's not what I have on my side. See below (from left to right: master, your PR and your PR with 0.95em)

![2021-02-19_20-50](https://user-images.githubusercontent.com/45535283/108554586-33b0b880-72ec-11eb-8193-7915c1c35e2d.png) 
![2021-02-19_20-52](https://user-images.githubusercontent.com/45535283/108554596-37443f80-72ec-11eb-8638-319ba4f38886.png) 
![2021-02-19_20-53](https://user-images.githubusercontent.com/45535283/108554605-3a3f3000-72ec-11eb-9794-b917738ad54c.png)

Reply:
1.05em is virtually indistinguishable from master at 9-12em so seems a good compromise

Reply:
@Nilvus what font size do you use

Reply:
here's what I see with font size 8.4

master, followed by my PR with 0.95 
![Screenshot_2021-02-19_20-19-54](https://user-images.githubusercontent.com/9555491/108557081-e8000e00-72ef-11eb-9b30-d41406b38db1.png)![Screenshot_2021-02-19_20-20-20](https://user-images.githubusercontent.com/9555491/108557086-eafafe80-72ef-11eb-9c39-e81181f27641.png)

Reply:
Forget all that. I really don't understand why I had UI shown in screenshots above, so issue I pointed here, but after removing completely the folder where I had the clone of your repository, I have now correct UI and can confirm that 1.15em is good. I suspect something not updated correctly in CSS file in source code. It's now ok. Sorry for the noise!

Reply:
That's ok. There's still some odd interaction here that I don't understand though, and I do notice a difference on larger fonts. The whole UI should scale evenly if everything is in ems. But changing the left/right margin shouldn't change the size of the icon when there's plenty of free space on either side of it. I'm assuming either there's something hard-coded in px somewhere or Gtk just isn't respecting the box model properly.